### DISCUSSION TITLE
Anúncio

### DISCUSSION THREAD

**Discussion by Rayanelet** [community/community#64660] [Programming Help]
### Body

Gente passando para avisar que criei um canal chamado RayaneCode , me sigam lá no YouTube, e no Instagram também que é o mesmo nome e me sigam aqui no GitHub!!
Me ajudem gente , qualquer dúvida só falar aqui 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by bronglil** [comment]
Claro, entendi! Parece que você criou um canal chamado "RayaneCode" no YouTube e também tem contas no Instagram e GitHub com o mesmo nome. Você está convidando as pessoas a seguirem você nessas plataformas para obterem conteúdo relacionado. É ótimo ver o seu entusiasmo! Boa sorte com o seu canal e atividades nas re

**Comment by Paulo-Ricard0** [comment]
Opa segui lá! se puder também marcar esse meu comentario como aceito para ajudar pfv

**Comment by queenofcorgis** [comment]
Hi @Rayanelet , we’re glad you’re here 😀

Thanks for sharing your socials with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
